azurerm/internal/services/network/tests/web_application_firewall_policy_resource_test.go

```diff
@@ -246,8 +252,11 @@ resource "azurerm_web_application_firewall_policy" "test" {
   }
 
   policy_settings {
-    enabled = true
-    mode    = "Detection"
+    enabled                     = true
+    mode                        = "Detection"
+    request_body_check          = true
+    file_upload_limit_in_mb     = 100
+    max_request_body_size_in_kb = 128
```

Comment:
Given these are the defaulted values, we should probably omit them.  If the API/service defaults change without our knowledge this will highlight it.